test/local/delete_account.sh

```
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # Load Environment vars
-source test/integration/test_envs 
+source test/integration/test_envs
```

Shellcheck (0.11.0)

[info] Not following: test/integration/test_envs was not specified as input (see shellcheck -x).

(SC1091)